profiles/arch/riscv/package.use.mask

```diff
@@ -3,7 +3,7 @@
 
 # Marek Szuba <marecki@gentoo.org> (2021-06-22)
 # Dependencies not keyworded here yet
-dev-python/pillow imagequant test tk
+dev-python/pillow tk
 
```

Comment:
should have separate commit for unmask USEs for pillow, comment it well
please remember it doesn't hurt to have more commits while using git ..